.all-contributors.md

```diff
@@ -0,0 +1,66 @@
+**Please ignore this file.**
+
+This list of contributors is updated automatically by [All Contributors](https://allcontributors.org/) and must not be used in production.
```

Comment:
I don't fully understand what we mean by “must not be used in production”. How am I supposed to “use in production” a Markdown file? 🙂

```suggestion
This list of contributors is generated automatically by [All Contributors](https://allcontributors.org/). The output used in production is the JSON format, this file is not used.
```

Reply:
It seems to me that we decided to say that because the markdown file could be used with a static site manager to create a page listing the contributors. But on rereading it, I find that it's useless.